mmv1/third_party/terraform/services/bigquery/resource_bigquery_dataset_iam_member_test.go

```diff
@@ -91,6 +126,38 @@ resource "google_service_account" "bqviewer" {
 `, datasetID, saID)
 }
 
+func testAccBigqueryDatasetIamMember_afterDatasetAccessCreation(projectID, datasetID, authDatasetID, saID string) string {
+	return fmt.Sprintf(`
+
+resource "google_bigquery_dataset" "auth_dataset" {
+	dataset_id = "%s"
+}
+
+resource "google_bigquery_dataset" "dataset" {
+  dataset_id = "%s"
+  access {
+    dataset {
+      dataset {
+        project_id = "%s"
+        dataset_id = google_bigquery_dataset.auth_dataset.dataset_id
+      }
+      target_types = ["VIEWS"]
+    }
+  }
+}
+
+resource "google_service_account" "bqviewer" {
+  account_id = "%s"
+}
+
+resource "google_bigquery_dataset_iam_member" "access" {
```

Comment:
Should we be checking the scenario where the IAM member is created first as well? Or is that covered elsewhere